vscode/src/notebook.ts

```diff
@@ -58,7 +57,6 @@ export function registerQSharpNotebookHandlers() {
             cell.document,
             qsharpLanguageId
           );
-          sendTelemetryEvent(EventType.QSharpJupyterCellInitialized);
```

Comment:
If this is the only place this eventtype was used, shouldn't you be removing the (now unused) definition from vscode/src/telemetry.ts also?